srcpkgs/python3-pytaglib/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'python3-pytaglib'
+pkgname=python3-pytaglib
+version=1.4.6
+revision=1
+wrksrc="pytaglib-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+makedepends="python3-devel taglib-devel"
+depends="taglib"
```

Comment:
Drop `taglib`, it is automatically detected.

Reply:
wait a sec I messed up

Reply:
ok fixed